named exports will cause Fast Refresh to not preserve local component state and do a full refresh

When doing a `gatsby develop` I get this warning:

```
warn 
/tmp/gatsby-starter-i18n-bulma/src/templates/blog-post.js
  10:1  warning  In page templates only a default export of a valid React component and the named export of a page
query is allowed.
        All other named exports will cause Fast Refresh to not preserve local component state and do a full refresh.

        Please move your other named exports to another file. Also make sure that you only export page queries that
use the "graphql" tag from "gatsby".
  limited-exports-page-templates
```

 I think this might be quite a performance hit:

https://www.gatsbyjs.com/docs/reference/local-development/fast-refresh/

I think this might be as simple as importing BlogPostTemplate from another file and Export  Blog Post elsewhere, but wanted to get thoughts on this from everyone else.

--- a/src/templates/blog-post.js
+++ b/src/templates/blog-post.js
@@ -7,7 +7,7 @@ import { graphql } from 'gatsby'
 import Layout from '../components/Layout'
 import Content, { HTMLContent } from '../components/Content'
 
-export const BlogPostTemplate = ({
+const BlogPostTemplate = ({
   data,
   location,
   content,

